packages/components/field-label/README.md

```diff
@@ -32,7 +32,7 @@ npm --save install react
 ```jsx
 import React from 'react';
 import { defineMessage, FormattedMessage } from 'react-intl';
-import { FlatButton } from '@commercetools-buttons/flat-button';
+import FlatButton from '@commercetools-uikit/flat-button';
```

Comment:
unrelated, but we did merge field-label yesterday so this can be included in the same release